src/beautifiers/sass-convert.coffee

```diff
@@ -0,0 +1,20 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class SassConvert extends Beautifier
+  name: "SassConvert"
+  link: "http://sass-lang.com/documentation/file.SASS_REFERENCE.html#syntax"
+
+  options:
+    # TODO: Add support for options
+    CSS: true
+    Sass: true
+    SCSS: true
```

Comment:
I have changed those options to `false` already. Sorry for missing that part.

Maybe a notice for running `npm run docs` should be added into the docs, since 8c77514 updated the docs much more than what I did :smile: